hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonReader.cpp

```diff
@@ -377,6 +375,10 @@ void OsmGeoJsonReader::_parseGeoJsonRelation(const string& id, const pt::ptree&
       bool use_old = _useDataSourceIds;
       _useDataSourceIds = true;
       pt::ptree geometries = geometry.get_child("geometries");
+      long points = 0,
+           polys = 0,
+           linestrings = 0,
+           other = 0;
```

Comment:
This is where the magic starts.  Keeping track of each type of geometry within the `GeometryCollection` for classification at the end.